### DISCUSSION TITLE
Latest LoadROI commit: information on that ROI file?

### DISCUSSION THREAD

**Discussion by urshess-siga** [cdcseacave/openMVS#1182] [Q&A]
In the latest commit, there is a new LoadROI feature, where we can hand over a ROI file.
Is there any information on the concept, or, more specifically, how to get such a ROI file or what it must contain.

I like the ROI feature and had some good results, had to tweak the threshold a bit though.
So I am very interested in any new possibilities here.

**Comment by cdcseacave** [comment]
LoadROI() is able to load either a AABB or OOB structure from a text file, in the format and logic defined in the two structures.

**Reply by KaiCiHuang** [reply]
Would you know those text how to arrange it ? I want to output OOB's text for OpenMVS.
